src/custom-elements/src/components/popover/popover.tsx

```diff
@@ -335,24 +335,15 @@ export class Popover {
   }
 
   connectedCallback() {
-    const observerTarget = this.el;
-    const mutationObserverConfig = {
-      attributes: true,
-      attributeOldValue: true,
-      attributeFilter: ['title'],
-    };
-
-    if (!this.mutationObserver) {
-      const subscriberCallback = (mutations) => {
-        mutations.forEach((mutation) => {
-          this.popoverTitle = mutation.target.title;
-        });
-      };
-
-      this.mutationObserver = new MutationObserver(subscriberCallback);
-    }
-
-    this.mutationObserver.observe(observerTarget, mutationObserverConfig);
+    addMutationObserver.call(this, () => {
+        this.popoverTitle = this.el.title;
```

Comment:
What this mutation does essentially is to set the attribute `title` to the property `popoverTitle` and  the same can be achieved using 'attribute: title' option in the prop decorator (https://stenciljs.com/docs/properties#attribute-name-attribute), bypassing the need for a mutation observer. Can we test if this covers all use cases for this component?

Reply:
After reviewing, the mutation observer should be used anyway (for childList and subtree) because the component has slots. For the title, the proposed solution works without mutation observer.